apps/docs/components/MDX/auth_error_codes_table.mdx

```diff
@@ -10,30 +10,40 @@ To supplement HTTP status codes, Supabase Auth returns a string error code which
 
 | Code                                    | Description                                                                                                                                                                                                                                                                                                   |
 | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Probably also need to add thse:

ErrorCodeUnexpectedFailure
ErrorCodeSAMLIdPAlreadyExists
ErrorCodeSAMLMetadataFetchFailed
ErrorCodeSAMLAssertionNoUserID
ErrorCodeSSOProviderNotFound
ErrorCodeSSODomainAlreadyExists
ErrorCodeSAMLEntityIDMismatch

Reply:
These are already in. Probably no need to include `unknown` is excluded since the user can't take action on it and it shouldn't be used directly